src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -491,31 +611,32 @@ public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
 
       if (archive) {
         new SnackbarAsyncTask<Long>(getView(),
-                                    getResources().getQuantityString(R.plurals.ConversationListFragment_moved_conversations_to_inbox, 1, 1),
-                                    getString(R.string.ConversationListFragment_undo),
-                                    getResources().getColor(R.color.amber_500),
-                                    Snackbar.LENGTH_LONG, false)
+                getResources().getString(R.string.ConversationListFragment_deleting),
+                "",
+                getResources().getColor(R.color.amber_500),
+                Snackbar.LENGTH_LONG, false)
+
         {
           @Override
           protected void executeAction(@Nullable Long parameter) {
-            DatabaseFactory.getThreadDatabase(getActivity()).unarchiveConversation(threadId);
+            DatabaseFactory.getThreadDatabase(getActivity()).deleteConversation(threadId);
           }
 
           @Override
           protected void reverseAction(@Nullable Long parameter) {
-            DatabaseFactory.getThreadDatabase(getActivity()).archiveConversation(threadId);
+            //unused
           }
         }.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR, threadId);
       } else {
         new SnackbarAsyncTask<Long>(getView(),
-                                    getResources().getQuantityString(R.plurals.ConversationListFragment_conversations_archived, 1, 1),
-                                    getString(R.string.ConversationListFragment_undo),
-                                    getResources().getColor(R.color.amber_500),
-                                    Snackbar.LENGTH_LONG, false)
+                getResources().getString(R.string.ConversationListFragment_deleting),
+                "",
```

Comment:
Same comment as above, is it standard to default to empty or is that not what that second parameter is?